universal-login-relayer/lib/utils/relayerUnderTest.js

```diff
@@ -2,13 +2,15 @@ import Relayer from '../relayer';
 import {defaultAccounts, getWallets} from 'ethereum-waffle';
 import ENSBuilder from 'ens-builder';
 import {withENS} from './utils';
+import knex from 'knex';
+import defaultKnexConfig from '../../knexfile';
 
 class RelayerUnderTest extends Relayer {
   url() {
     return `http://127.0.0.1:${this.port}`;
   }
 
-  static async createPreconfigured(provider) {
+  static async createPreconfigured(provider, knexConfig = defaultKnexConfig) {
```

Comment:
Now you can use: `knexConfig = getKnexConfig()`, which will include environment (always test)

Reply:
✅